src/appsignal/config.py

```diff
@@ -104,6 +108,38 @@ class Options(TypedDict, total=False):
 )
 
 
+def validate(config: Options):
+    config["log_file_path"] = log_file_path(config)
+
+
+def log_file_path(config: Options) -> Optional[str]:
+    filename = "appsignal.log"
+    path = config.get("log_path")
```

Comment:
I recall other integrations remove `appsignal.log` from the end of the path if already present. Should we do that here as well?

Reply:
That was because at some point we told people to specify it. If we never tell people that for Pythong, we never support it. But we can choose to remove it anyway, so you don't get `/path/to/log/mylog.logappsignal.log` kind of scenario, but that already wouldn't work in most cases because `/path/to/log/mylog.log` probably doesn't exist and won't be writable.

Reply:
Added a commit that removes a filename if it's present.